api/src/test/java/org/pmiops/workbench/utils/mappers/NewUserSatisfactionSurveyMapperTest.java

```diff
@@ -0,0 +1,47 @@
+package org.pmiops.workbench.utils.mappers;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.pmiops.workbench.db.model.DbNewUserSatisfactionSurvey;
+import org.pmiops.workbench.db.model.DbNewUserSatisfactionSurvey.Satisfaction;
+import org.pmiops.workbench.db.model.DbUser;
+import org.pmiops.workbench.model.CreateNewUserSatisfactionSurvey;
+import org.pmiops.workbench.model.CreateNewUserSatisfactionSurvey.SatisfactionEnum;
+import org.pmiops.workbench.survey.NewUserSatisfactionSurveyMapper;
+import org.pmiops.workbench.survey.NewUserSatisfactionSurveyMapperImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+@Import(NewUserSatisfactionSurveyMapperImpl.class)
+@SpringJUnitConfig
+public class NewUserSatisfactionSurveyMapperTest {
+  @Autowired private NewUserSatisfactionSurveyMapper mapper;
+
+  @Test
+  public void testToDbNewUserSatisfactionSurvey() {
+    final DbUser user = new DbUser();
+    final String additionalInfo = "I love the app";
+
+    CreateNewUserSatisfactionSurvey createNewUserSatisfactionSurvey =
+        new CreateNewUserSatisfactionSurvey();
+    createNewUserSatisfactionSurvey.setSatisfaction(SatisfactionEnum.SATISFIED);
+    createNewUserSatisfactionSurvey.setAdditionalInfo(additionalInfo);
+    DbNewUserSatisfactionSurvey mappedDbNewUserSatisfactionSurvey =
+        mapper.toDbNewUserSatisfactionSurvey(createNewUserSatisfactionSurvey, user);
+
+    DbNewUserSatisfactionSurvey dbNewUserSatisfactionSurvey =
+        new DbNewUserSatisfactionSurvey()
+            .setSatisfaction(Satisfaction.SATISFIED)
+            .setAdditionalInfo(additionalInfo)
+            .setUser(user);
+
+    assertThat(mappedDbNewUserSatisfactionSurvey.getSatisfaction())
+        .isEqualTo(dbNewUserSatisfactionSurvey.getSatisfaction());
```

Comment:
I'd prefer to see `createNewUserSatisfactionSurvey` for these assertions, to show that the mapping preserved the original's fields more directly.

Reply:
Makes sense! It's a `CreateNewUserSatisfactionSurvey` -> `DbNewUserSatisfactionSurvey` after all. Updated.